### PR TITLE
change list, sone lines

### PR DESCRIPTION
**Pull request by Hoper**

### CODE REVIEW

**Comment by Hoper** [comment]
ok

File: list.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
 
-543
+123
```
Comment (Hoper) [inline list.html:2]: good idea

**Review by Hoper — COMMENTED** [review]
ok

File: list.html (hunk 1 above)
Comment (DenisPlastinin) [reply list.html:2]: yes

**Review by DenisPlastinin — APPROVED** [review]

**Pull request merged**